mithril-common/src/digesters/immutable_digester.rs

```diff
@@ -55,12 +52,10 @@ pub enum ImmutableDigesterError {
 
     /// Error raised when there's less than the required number of completed immutables in
     /// the cardano database or even no immutable at all.
-    #[error("At least two immutables chunk should exists")]
+    #[error("At least two immutables chunk should exists in directory '{db_dir}'.")]
```

Comment:
You can use `{found_number:?}` in this error message as `found_number` is an option